openbb_terminal/stocks/screener/finviz_model.py

```diff
@@ -188,9 +188,7 @@ def get_screener_data(
             else:
                 df_screen = screen.screener_view(ascend=ascend)
 
-    df_screen.rename(
-        columns={"\n\nTicker": "Ticker", "Ticker\n\n": "Ticker"}, inplace=True
-    )
+    df_screen.columns = [val.strip("\n") for val in df_screen.columns]
```

Comment:
Fun fact: `.strip()` will get rid of all newlines as well as leading and trailing whitespaces